### ISSUE TITLE
Add relative url in store collection.

### ISSUE THREAD

**Issue by vnmone** [jalik/jalik-ufs#78]
I think, it will be better to store url, relative to ROOT_URL.
Now url stores as absolute url, and i cant transfer files from dev to production.

**Comment by jalik** [comment]
You have a specific case... you should be able to migrate your files easily with this code (adapt it) :
```js
// on server
Meteor.startup(()=> {
    Files.find({}).forEach((file)=> {
        // This updates the file URL using the current value of ROOT_URL
        Files.update({_id: file._id}, {$set: {url: FileStore.getFileURL(file._id)}});
    });
});
```

**Comment by vnmone** [comment]
thanks! I missed getFileUrl() method.
but what is the reason to save fullpath?

**Comment by jalik** [comment]
@vnmone, there is not real reason but having an absolute URL which is more reliable when embedding links in emails or things like that, you could do that manually. I could maybe add a `path` attribute to uploaded files. But at the end it's not a big deal as you won't change your domain every months and the migration is easy as you saw. Also I was trying to keep the same behavior as **CollectionFS** to facilitate any migration.

**Comment by jalik** [comment]
Fixed in the [v0.6.8](https://github.com/jalik/jalik-ufs#version-068), use the `path` attribute to get the relative file URL.
You can migrate existing files by calling `UploadFS.addPathAttributeToFiles()` on the server.
